airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -252,6 +251,18 @@ class Contacts(IncrementalIntercomStream):
     Endpoint: https://api.intercom.io/contacts
     """
 
+    @staticmethod
```

Comment:
As @sherifnada has suggested is there a reason for making this methods static?

Reply:
Updated to not static.